deploy/PortabilityService.ServiceFabricMeshApplication/PortabilityService-1709.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "eastus",
+      "metadata": {
+        "description": "Location of the resources."
+      }
+    },
+    "registry-server": {
+      "type": "string",
+      "defaultValue": "portabilityservice.azurecr.io",
```

Comment:
Should this be public?

Reply:
Good question. I don't think there's much danger including the registry name and user name as defaults since the password still needs to be provided. It may be better to not include them, though, just because they aren't useful to anyone without access to this particular registry.